### DISCUSSION TITLE
Expose a health check

### DISCUSSION THREAD

**Discussion by Meemaw** [storybookjs/storybook#23729] [Ideas]
### Is your feature request related to a problem? Please describe.

When trying to use Storybook for visual tests (e.g using Playwright), we want to start the storybook server and wait for it to be ready.

This can be done with something like this:
```js
npx wait-on http-get://127.0.0.1:6006
```

This works, however this will pass as soon as the server is up, but when trying to visit a story, it will start compiling modules and take a very long time (depending on the project size).

When visiting a story with Playwright this will result in a timeout.

### Describe the solution you'd like

It would be great if Storybook exposed an endpoint that could be used as a signal that modules are compiled & everything is ready to be used.

### Are you able to assist to bring the feature to reality?

no

**Comment by haslers** [comment]
We had the same problem in our project and found a workaround: 
- build storybook to static files first (`yarn build-storybook`)
- serve them with e.g. http-server on your desired port and start the tests
